lint-staged.config.mjs

```diff
@@ -5,7 +5,8 @@ export default {
 	"*.{ts,tsx,js,cts,cjs,mts,mjs}": [
 		"pnpm run lint:js",
 		// ignore staged file list; execute x without extra args
-		() => "pnpm run x ae ci"
+		() => "pnpm run x ae update",
+		"git add ."
```

Comment:
This may add unexpected files

Reply:
Going to revert this. It's used for local pushing, which should be removed at the end. 🥲